be/src/common/config.h

```diff
@@ -759,6 +758,10 @@ CONF_String(starlet_cache_dir, "");
 
 CONF_Int64(lake_metadata_cache_limit, /*2GB=*/"2147483648");
 
-} // namespace config
+CONF_mBool(enable_librdkafka_debug_mode, "false");
 
-} // namespace starrocks
+// generic, broker, topic, metadata, feature, queue, msg, protocol, cgrp, security
```

Comment:
what's the mearning of these words?

Reply:
"A comma-separated list of debug contexts to enable. Detailed Producer debugging: broker,topic,msg. Consumer: consumer,cgrp,topic,fetch",

Reply:
Already add the detailed comments;